src/value_deserializer.rs

```diff
@@ -28,35 +31,36 @@ pub struct CxxValueDeserializerDelegate {
 
 #[no_mangle]
 pub unsafe extern "C" fn v8__ValueDeserializer__Delegate__ReadHostObject(
-  this: &mut CxxValueDeserializerDelegate,
-  _isolate: *mut Isolate,
+  this: &CxxValueDeserializerDelegate,
+  isolate: *mut Isolate,
 ) -> *const Object {
-  let value_deserializer_heap = ValueDeserializerHeap::dispatch_mut(this);
-  let scope =
-    &mut crate::scope::CallbackScope::new(value_deserializer_heap.context);
-  let value_deserializer_impl =
-    value_deserializer_heap.value_deserializer_impl.as_mut();
-  match value_deserializer_impl.read_host_object(
-    scope,
-    &mut value_deserializer_heap.cxx_value_deserializer,
-  ) {
+  let value_deserializer_heap = ValueDeserializerHeap::dispatch(this);
+  let scope = &mut CallbackScope::new(isolate.as_mut().unwrap());
```

Comment:
without the cppgc migration i think you can undo the changes here

Reply:
I think it's still needed, since you need to get a `Local<Context>` from a `Global<Context>`